app/views/shared/_github_button.html.erb

```diff
@@ -1,9 +1,9 @@
 <% if @user.password_required? %>
   <% resource_class.omniauth_providers.each do |provider| %>
     <div class="omniauth-login text-center">
-      <%= link_to omniauth_authorize_path(resource_name, provider), class: 'github-login-button page-buttons login btn' do %>
```

Comment:
Just to be sure, this partial is only called from the login and sign up pages yes? Can anyone confirm?